app/views/layouts/_sub_sub_menu.html.erb

```diff
@@ -66,6 +66,12 @@
               </li>
             <% end %>
           <% end %>
+          <% if @lti_deployments %>
+          <li class='<%= "active" if controller.action_name == 'create_lti_line_items' %>'>
+            <%= link_to I18n.t('lti.lti_settings'),
+                        create_lti_line_items_course_assignment_path(@current_course, @assignment) %>
```

Comment:
Okay so overall I like the idea of creating a new tab. However, for this tab please create a new assignment route called `lti_settings` to render the page. Right now that page can have a title and just include the (renamed) `_create_lti_line_items.html.erb` as a partial, but we'll likely expand on this later.

Then, you can ensure the `create_lti_line_items` route is just a POST, not a GET.